packages/vc-verification/src/resolver/Ipfs-credential-resolver.ts

```diff
@@ -108,11 +108,13 @@ export class IpfsCredentialResolver implements CredentialResolver {
     namespace: string
   ): Promise<RoleEIP191JWT | undefined> {
     const eip191Jwts = await this.eip191JwtsOf(did);
-    return eip191Jwts.find(
-      (jwt) =>
-        jwt?.payload?.claimData.claimType === namespace ||
-        utils.namehash(jwt?.payload?.claimData.claimType) === namespace
-    );
+    return eip191Jwts
+      .filter((jwt) => typeof jwt.payload.claimData.claimType === 'string')
```

Comment:
@JGiter Thanks for suggesting the fix but I don't really get it. Could you provide an example of a jwt for which `jwt.payload.claimData.claimType === 'string'` isn't a string? I think the best way to do this would be to add a test for it.

Reply:
This check mainly supposed to ensure that `claimType` is defined. I'll add test for this